src/renderer/src/pages/home/index.tsx

```
@@ -1,11 +1,12 @@
-import Chart from '@renderer/components/home/Chart/Chart'
+import { Chart, Series } from '@renderer/components/home'
 import { flex } from '@renderer/utils'
 import { styled } from 'styled-components'
 
 const Home = () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)